cli/admin/src/main/java/io/pravega/cli/admin/dataRecovery/RecoverFromStorageCommand.java

```diff
@@ -657,4 +655,27 @@ public void close() {
         }
     }
 
+    private void validateArguments() {
+        Preconditions.checkArgument(getArgCount() >= 2, "Incorrect argument count.");
+        final String container = getArg(1);
+
+        if (!NumberUtils.isNumber(container)) {
+            Preconditions.checkArgument(container.toLowerCase().equals("all"), "Container argument should either be ALL/all or a container id.");
+            Preconditions.checkArgument(getArgCount() >= 2, "Incorrect argument count.");
```

Comment:
isn't this already handled at line no `659` ?

Reply:
Correct, that was a mistake it should be equal to 2 as when we pass all then we don't need the end container parameter. Corrected it.